data/builder/file_test.go

```diff
@@ -24,33 +24,33 @@ var referenceTestCases = []struct {
 }{
 	{
 		size:            1024,
-		bareExpected:    cid.MustParse("bafkreibhn6gptq26tcez7zjklms4242pmhpiuql62ua2wlihyxdxua2nsa"),
-		wrappedExpected: cid.MustParse("bafybeig6rdrdonmqxao32uihcnnbief4qdrjg4aktfo5fmu4cdeqya3glm"),
+		bareExpected:    cid.MustParse("bafkreigwqvgm5f6vgdv7wjkttdhgnkpbazhvuzvrqzaje4scb4moeinjum"),
```

Comment:
why do we get new cids here but not elsewhere?

Reply:
Different random numbers here because `ipfsutil.NewSeededRand` uses `Rand.Intn(255)` for each random byte [here](https://github.com/ipfs/go-ipfs-util/blob/master/util.go#L98), and `random.NewSeededRand` uses the `Read` function directly from the `rand.Rand` instance. Not elsewhere, because specific numbers are not being expected elsewhere, so the actual generated numbers do not matter so long as whatever number is generated is used/stored/communicated correctly.

Reply:
ok, as long as this is expected!